continuous_integration/environment-3.12.yaml

```diff
@@ -35,10 +35,10 @@ dependencies:
   # https://github.com/dask/dask/pull/9569
   # - tiledb-py # crashes on Python 3.11
   # - pyspark
-  # - tiledb>=2.5.0 # crashes on Python 3.11
+  # - tiledb # crashes on Python 3.11
   - xarray
-  - sqlalchemy>=1.4.16
-  - pyarrow>=13
+  - sqlalchemy
+  - pyarrow
```

Comment:
Please keep the arrow versions restricted to ensure that we test things like the convert string options

Reply:
OK, I've changed the 3.11 recipe from `pyarrow>=11` to `pyarrow>=13`. The former IMHO caused confusion as the recipe installs 13 anyway.

Reply:
Fine with me as long as we have a lower pin that forces the string option